b9lab-content/3-main-concepts/16-ibc.md

```diff
@@ -8,4 +8,4 @@ Now we want to let players choose to play with foreign tokens. So:
 * The _billing_ needs to be updated.
 * The leaderboard does not need to be adjusted because it is already per token type. We don't want to do another migration example, that's why we had to be clever about the leaderboard structure.
 
-TODO decide how it looks like. 
+TODO decide how it looks like.
```

Comment:
What's this TODO about?

Reply:
It is about the code samples. You can discard this comment.